be/src/main/java/yeonba/be/user/repository/favorite/FavoriteQuery.java

```diff
@@ -0,0 +1,18 @@
+package yeonba.be.user.repository.favorite;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import yeonba.be.user.entity.User;
+
+@Component
+@RequiredArgsConstructor
+public class FavoriteQuery {
+
+  private final FavoriteRepository favoriteRepository;
+
+  public boolean existsByUserAndFavoriteUser(User user, User favoriteUser){
```

Comment:
지난 PR에서 함수명에 대해서 얘기나눴는데 해당 함수명이 적합하다고 생각을 하신건가요?
`alreadyRegisted~~`와 같은 함수명은 어떤가요?
```suggestion

  public boolean alreadyRegistedFavoriteBetween(User user, User favoriteUser) {
```
예를 든 이름입니다. 생각해서 적절한 함수명을 사용하시면 좋겠습니다. + 코드 컨벤션 생각해주세요.

Reply:
확인했습니다. 수정해서 반영하도록 하겠습니다.